jax/numpy/lax_numpy.py

```diff
@@ -493,6 +493,16 @@ def flip(m, axis):
   return lax.rev(m, [axis])
 
 
+@_wraps(onp.fliplr)
+def fliplr(m):
+  return flip(m, 1)
```

Comment:
What if the array has rank > 2? I think these need to be phrased as -1, -2.
However, I think this also points to a bug in the implementation of `flip` — it needs to support negative axes.

https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.fliplr.html

You could also write these two as:
`m[..., ::-1]`
and
`m[..., ::-1, :]`

but the problem with `flip` still remains.

Reply:
Numpy flipud works on the first axis, not the last: https://github.com/numpy/numpy/blob/v1.15.1/numpy/lib/twodim_base.py#L137

Reply:
Huh. You're right. The docs are actually a bit misleading.

The problem with `flip` not accepting negative axes is real.

Please add a 3D test.

Reply:
Added support for negative indices, but when I tried to support None, ran into deeper issues.  `None` behavior and setting it as a default was just added in numpy 1.15.  Presumably its alright to leave that off?